### ISSUE TITLE
.meta-file as XML-file

### ISSUE THREAD

**Issue by CarolinOdebrecht** [korpling/pepperModules-GenericXMLModules#4]
There is a resourceException whenn converting an XML-doc to annis and using the property <property key="pepper.before.readMeta">meta</property>  in the pepper workflow. (Minimal example 1 attached)

When providing metadata in XML (e.g. <corpus>Test</corpus>), there is no resourceException (Minimal example 2 attached). I think, the .meta file is imported as a corpusfile in example 1, and in example 2 the output is even more confusing. We got a document for .meta-file and metadata.
[XMLImporterTest1.zip](https://github.com/korpling/pepperModules-GenericXMLModules/files/821547/XMLImporterTest1.zip)
[XMLImporterTest2.zip](https://github.com/korpling/pepperModules-GenericXMLModules/files/821549/XMLImporterTest2.zip)

**Comment by FlorianZipser** [comment]
You are absolutely right, there is a bug in reading files, which are no xml files. Thanks for your report. Unfortunately I am not sure if this is easily fixable, but I can give you a workaround. You can avoid the parsing of non xml files by filtering the file ending if this is possible in your case. For instance for the sample corpus you have attached, you can add the following property to the GenericXMLImporter in the Pepper workflow file:
```
<importer name="GenericXMLImporter" path="./XML">
   ...
   <property key="genericXml.importer.file.endings">xml</property>
</importer>
```
